### ISSUE TITLE
[Task]: Add newsletter call-to-action to "Homepage > Goals" section

### ISSUE THREAD

**Issue by andycochran** [HHS/simpler-grants-gov#712]
### Summary

**BLOCKED: ~#708~ #789 ** 

[Figma prototype](https://www.figma.com/proto/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=918-1623&t=nFLaS0HsocPUQI8k-0&scaling=scale-down-width&page-id=918%3A1622&starting-point-node-id=918%3A1623)

<img width="628" alt="image" src="https://github.com/HHS/simpler-grants-gov/assets/111455374/9b40b35d-106b-40da-86dd-94c86e675f5b">

### Acceptance criteria

- [ ]  Add the newsletter CTA `simpler.grants.gov/newsletter` to the homepage in the Goals section: 
- [ ] Tests pass and there is a storybook component
- [ ] Design will need to be responsive

**Comment by andycochran** [comment]
I don't think this is blocked by #708. This just needs to link to the /newsletter page. It's just blocked by creating that page.

**Comment by acouch** [comment]
This has been superseded by #827
